engine/src/deploy.cpp

```diff
@@ -801,7 +801,7 @@ void MCIdeDeploy::exec_ctxt(MCExecContext& ctxt)
 
 	// If the banner_class field is set and we are not a trial license, we
 	// override the license class with that specified.
-	uint32_t t_license_class;
+	MCLicenseClass t_license_class;
 	t_license_class = kMCLicenseClassNone;
```

Comment:
Again, if you're tweaking this, might as well declare and intiialise on the same line